server/src/db.ts

```diff
@@ -0,0 +1,16 @@
+import { MongoClient } from "mongodb";
+
+const db = async (config: Config) => {
+  const mongoClient = new MongoClient(config.provenDb.uri);
+
+  try {
+    await mongoClient.connect();
+    const db = mongoClient.db();
+    const currentVersion = await db.command({ getVersion: 1 });
+    console.log("connecting to proven db at" + JSON.stringify(currentVersion));
```

Comment:
Not nitpicking but... 

```suggestion
    console.log("connecting to proven db at " + JSON.stringify(currentVersion));
```